CHANGELOG.md

```diff
@@ -68,6 +68,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1695](https://github.com/open-telemetry/opentelemetry-python/pull/1695))
 - Change Jaeger exporters to obtain service.name from span
   ([#1703](https://github.com/open-telemetry/opentelemetry-python/pull/1703))
+- Fixed a parse error on no `OTEL_TRACES_EXPORTER`
```

Comment:
can you change the description to be focused on the behavior of the user?

In this case, I would expect something like `Fixed an unset OTEL_TRACES_EXPORTER resulting in an error`.

This way a reader can identify whether their behavior is fixed or not.